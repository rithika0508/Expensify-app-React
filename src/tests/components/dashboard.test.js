import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/dashboard';
import toJSON from 'enzyme-to-json';

test('should render header correctly', () => {
    const wrapper = shallow(<ExpenseDashboardPage />)
    expect(toJSON(wrapper)).toMatchSnapshot();

    // const renderer = new ReactShallowRenderer()
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
})