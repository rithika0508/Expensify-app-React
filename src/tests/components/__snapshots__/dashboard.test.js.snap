// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div>
  <Connect(ExpenseFilters) />
  <Connect(AllExpenses) />
</div>
`;
